packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,230 @@
+    this._awsContext = options.context;
+    this._timeoutWarning = options.timeoutWarning;
+    this._flushTime = options.flushTime;
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public setupOnce(_: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
+    const lambdaBootstrap: any = require.main;
+
+    /** configured time to timeout error and calculate execution time */
+    const configuredTimeInMilliseconds = this._awsContext.getRemainingTimeInMillis();
+
+    if (!this._awsContext && !lambdaBootstrap) {
+      return;
+    }
+    const processEnv: any = process.env;
+    /** rapid runtime instance */
+    const rapidRuntime = lambdaBootstrap.children[0].exports;
+
+    /** handler that is invoked in case of unhandled and handled exception */
+    const originalPostInvocationError = rapidRuntime.prototype.postInvocationError;
+
+    const hub = getCurrentHub && getCurrentHub();
+
+    /**
+     * This function sets Additional Runtime Data which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    const setAdditionalRuntimeData = (scope: Scope): void => {
+      scope.setContext('Runtime', {
+        Name: 'node',
+        // global.process.version return the version of node
+        Version: global.process.version,
+      });
+    };
+
+    /**
+     * This function sets Additional Lambda Parameters which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    const setAdditionalLambdaParameters = (scope: Scope): void => {
+      const remainingTimeInMillisecond: number = this._awsContext.getRemainingTimeInMillis();
+      const executionTime: number = configuredTimeInMilliseconds - remainingTimeInMillisecond;
+
+      scope.setExtra('lambda', {
+        aws_request_id: this._awsContext.awsRequestId,
+        function_name: this._awsContext.functionName,
+        function_version: this._awsContext.functionVersion,
+        invoked_function_arn: this._awsContext.invokedFunctionArn,
+        execution_duration_in_millis: executionTime,
+        remaining_time_in_millis: this._awsContext.getRemainingTimeInMillis(),
+      });
+    };
+
+    /**
+     * This function use to generate cloud watch url
+     */
+    const cloudwatchUrl = (): string => {
+      /**
+       * processEnv.AWS_REGION -  this parameters given the AWS region
+       * processEnv.AWS_LAMBDA_FUNCTION_NAME - this parameter provides the AWS Lambda Function name
+       */
+      return `https://${processEnv.AWS_REGION}.console.aws.amazon.com/cloudwatch/home?region=${processEnv.AWS_REGION}#logsV2:log-groups/log-group/$252Faws$252Flambda$252F${processEnv.AWS_LAMBDA_FUNCTION_NAME}`;
+    };
+
+    /**
+     * This function sets Cloud Watch Logs data which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    const setCloudwatchLogsData = (scope: Scope): void => {
```

Comment:
Same here

Reply:
Made this change in the new Pull Request. At the other 2 places, the setExtra is not updated as it was not setting 'sys-argv' parameter properly.